Libraries/Components/StatusBar/StatusBar.js

```diff
@@ -185,6 +264,60 @@ class StatusBar extends React.Component<{
    */
   static currentHeight = StatusBarManager.HEIGHT;
 
+  /**
+   * Get the current frame (height and y offset) of the Status Bar.
+   *
+   * Returns a Promise that resolves to a frame object with a `y` and `height`.
+   */
+  static getCurrentFrame(): Promise<{y: number, height: number}> {
+    return StatusBarManager.getCurrentFrame();
+  }
+
+  /**
+   * Adds an event handler. Supported events:
+   *
+   * - `statusBarFrameDidChange`: Fires when the status bar frame has changed.
+   *   The argument to the event handler is a frame object with the keys:
+   *   - `height`: The current height of the status bar.
+   *   - `y`: The y offset of the status bar, if any. This generally only happens
+   *     when the iOS In-Call Status Bar is visible and pushes the app down
+   *     instead of changing the size.
+   */
+  static addEventListener(
```

Comment:
Why do we introduce the new listeners APIs (with `remove` function) here?

Reply:
This is how NetInfo, AppState, and most of the classes that have events do it, so I just copied that. What is the best practice for new classes with events?